core/frontend/src/extension/providers/ServiceExtensionProvider.ts

```diff
@@ -20,10 +21,10 @@ import type { AccessToken } from "@itwin/core-bentley";
 export interface ServiceExtensionProviderProps {
   /** Name of the uploaded extension */
   name: string;
```

Comment:
I think the getAccessToken is still needed to allow other configure a separate authority from the one configured on `IModelApp`.

Reply:
getAccessToken() is still a prop in ServiceExtensionProviderProps, do you mean it shouldn't be internal?

Reply:
huh for some reason my diff had shown it was deleted... weird